modules/end-user-guide/pages/importing-the-source-code-of-a-project-into-a-workspace.adoc

```diff
@@ -0,0 +1,6 @@
+[id="importing-the-source-code-of-a-project-into-a-workspace"]
+:navtitle: Importing the source code of a project into a workspace
+:keywords: end-user-guide, importing-the-source-code-of-a-project-into-a-workspace
+:page-aliases: .:importing-the-source-code-of-a-project-into-a-workspace, importing-the-source-code-of-a-project-into-a-workspace
```

Comment:
```suggestion
:page-aliases: .:importing-the-source-code-of-a-project-into-a-workspace, importing-the-source-code-of-a-project-into-a-workspace, .:creating-a-workspace-by-importing-the-source-code-of-a-project, creating-a-workspace-by-importing-the-source-code-of-a-project
```

Reply:
When renaming pages, add page-aliases to handle redirections from the older page name.